src/items.cpp

```diff
@@ -2439,3 +2439,145 @@ void createCustomInventory(Stat* stats, int itemLimit)
 		}
 	}
 }
+
+node_t* itemNodeInInventory(Stat* myStats, Item* itemToFind, Category cat)
```

Comment:
Or I guess not for this specific case, seeing as `itemToFind` was changed to a non-pointer type. Regardless, in general, this applies. If a function needs an argument, said argument usually should be a reference.